### PR TITLE
V6/refactor access

### PR DESCRIPTION
**Pull request by olgakup**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label

### CODE REVIEW

File: src/core/router/routes-default.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ import ThePressKitLayout from '@/views/layouts-default/ThePressKitLayout';
...
-import MobileApps from '@/modules/wallets/pages/access/mobile-apps/MobileApps';
-import BrowserExtension from '@/modules/wallets/pages/access/browser-extension/BrowserExtension';
-import BrowserExtensionInstall from '@/modules/wallets/pages/access/browser-extension/install-extension/InstallExtension';
-import BrowserExtensionAccess from '@/modules/wallets/pages/access/browser-extension/access-extension/AccessExtension';
+import TheAccessWalletLayout from '@/views/layouts-default/TheAccessWalletLayout';
+// import HardwareWallets from '@/modules/wallets/pages/access/hardware-wallets/HardwareWallets';
+// import MobileApps from '@/modules/wallets/pages/access/mobile-apps/MobileApps';
+// import BrowserExtension from '@/modules/wallets/pages/access/browser-extension/BrowserExtension';
+// import BrowserExtensionInstall from '@/modules/wallets/pages/access/browser-extension/install-extension/InstallExtension';
+// import BrowserExtensionAccess from '@/modules/wallets/pages/access/browser-extension/access-extension/AccessExtension';
```
Comment (kvhnuke) [inline src/core/router/routes-default.js:15]: commented code

File: src/core/router/routes-default.js [hunk 2]
```diff
@@ -110,54 +110,54 @@ export default {
...
+    //   }
+    // },
+    // {
+    //   path: 'wallet/access/mobile-apps',
+    //   name: 'MobileApps',
+    //   component: MobileApps,
+    //   meta: {
+    //     requiresAuth: false
+    //   }
+    // }
```
Comment (kvhnuke) [inline src/core/router/routes-default.js:161]: commented code

File: src/modules/access-wallet/ModuleAccessWalletSoftware.vue [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+        url: 'https://kb.myetherwallet.com/en/offline/using-mew-offline'
+      },
+      buttons: [
+        /* Keystore Button */
+        {
+          label: 'Keystore',
+          description: 'Access via Keystore',
+          icon: require('@/assets/images/icons/icon-keystore-file.svg'),
+          fn: () => {
+            this.setType(SOFTWARE_WALLET_TYPES[0]);
```
Comment (kvhnuke) [inline src/modules/access-wallet/ModuleAccessWalletSoftware.vue:130]: lets switch this to Object so itll be `SOFTWARE_WALLET_TYPES.KEYSTORE`

File: src/modules/access-wallet/software/handlers/helpers.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { isHexString } from 'ethereumjs-util';
+
+const SOFTWARE_WALLET_TYPES = [
+  'keystore',
+  'mnemonic',
+  'private-key',
+  'overview'
+];
```
Comment (kvhnuke) [inline src/modules/access-wallet/software/handlers/helpers.js:8]:
```
SOFTWARE_WALLET_TYPES = {
KEYSTORE:'keystore'....
}
```

File: src/views/layouts-default/TheAccessWalletLayout.vue [hunk 5]
```diff
@@ -126,30 +162,71 @@ export default {
...
           title: 'Software',
           subtitle: 'Keystore files, Mnemonic phrase, Private key',
-          note: 'NOT RECOMMANDED',
+          note: 'NOT RECOMMENDED',
           rightIcon: '',
           titleIcon: 'mdi-alert',
           titleIconClass: 'warning--text text--darken-1',
           fn: () => {
-            this.open('showSoftware');
+            this.open(VALID_OVERLAYS[2]);
```
Comment (kvhnuke) [inline src/views/layouts-default/TheAccessWalletLayout.vue:175]: same here, lets switch it to an Object

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/views/layouts-default/TheAccessWalletLayout.vue (hunk 5 above)
Comment (olgakup) [reply src/views/layouts-default/TheAccessWalletLayout.vue]: fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2846/commits/2f678616ff35a0e9dccfbdce859132cf5e069d58

File: src/modules/access-wallet/software/handlers/helpers.js (hunk 4 above)
Comment (olgakup) [reply src/modules/access-wallet/software/handlers/helpers.js]: fixed in : https://github.com/MyEtherWallet/MyEtherWallet/pull/2846/commits/2f678616ff35a0e9dccfbdce859132cf5e069d58

File: src/modules/access-wallet/ModuleAccessWalletSoftware.vue (hunk 3 above)
Comment (olgakup) [reply src/modules/access-wallet/ModuleAccessWalletSoftware.vue]: fixed in https://github.com/MyEtherWallet/MyEtherWallet/pull/2846/commits/2f678616ff35a0e9dccfbdce859132cf5e069d58

**Review by gamalielhere — APPROVED** [review]

File: src/modules/access-wallet/software/handlers/helpers.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import { isHexString } from 'ethereumjs-util';
+
+const SOFTWARE_WALLET_TYPES = {
+  KEYSTORE: 'keystore',
+  MNEMONIC: 'mnemonic',
+  PRIVATE_KEY: 'private-key',
+  OVERVIEW: 'overview'
+};
```
Comment (kvhnuke) [inline src/modules/access-wallet/software/handlers/helpers.js:8]: lets import these from `configWalletTypes`

**Review by kvhnuke — APPROVED** [review]

File: src/modules/access-wallet/software/handlers/helpers.js (hunk 6 above)
Comment (olgakup) [reply src/modules/access-wallet/software/handlers/helpers.js:8]: These are actually different they are used in the route query definitions. overview does not exists in the configWalletTypes.

File: package.json [hunk 7]
```diff
@@ -16,7 +16,7 @@
     "lint:md": "remark . -o -f",
     "lint:styles": "prettier --write 'src/**/*.{scss,css}' && stylelint 'src/**/*.{scss,css}' --fix",
     "lint:tests": "eslint --fix tests/",
-    "preinstall": "find node_modules -name \".git\" -exec rm -rf {} \\; || true && bash checklockconflicts.sh && npx npm-force-resolutions",
```
Comment (gamalielhere) [inline package.json:19]: Add back force resolutions

File: src/core/router/routes-default.js [hunk 8]
```diff
@@ -110,54 +105,54 @@ export default {
...
+    //     requiresAuth: false
+    //   }
+    // },
+    // {
+    //   path: 'wallet/access/mobile-apps',
+    //   name: 'MobileApps',
+    //   component: MobileApps,
+    //   meta: {
+    //     requiresAuth: false
+    //   }
```
Comment (gamalielhere) [inline src/core/router/routes-default.js:155]: can probably just remove these

**Review by gamalielhere — APPROVED** [review]

File: package.json (hunk 7 above)
Comment (olgakup) [reply package.json]: fixed:  b88e914

File: src/core/router/routes-default.js (hunk 8 above)
Comment (olgakup) [reply src/core/router/routes-default.js]: fixed f904c6c

**Review by kvhnuke — APPROVED** [review]

**Pull request merged**